### ISSUE TITLE
Beam Advisor is still invisible even after setting the visibity to "visible."

### ISSUE THREAD

**Issue by Acemcbutterpants** [ThatTonybo/BeamAdvisor#9]
Okay.
Beam Advisor is invisible for me. Easy fix, right? Just go into the dev settings with Ctrl+U, and set beamAdvisor-visibility to visible. Then alt+f4.
I changed the setting. I quit the game to save it. I restarted the game. It did nothing.
I reinstalled the mod, changed the setting, quit the game, started the game, no difference.
I have no UI mods, and the only mod I could imagine would cause problems is BeamMP, but I don't know why it would break Beam Advisor.
Here's a screenshot. I don't know if the error message is related, so you can figure it out.
![image](https://user-images.githubusercontent.com/108896392/177850116-c6ddc832-9f36-4e4b-8b27-54fc940dc16f.png)
Please help.

**Comment by Acemcbutterpants** [comment]
(Ignore my messy UI)

**Comment by ThatTonybo** [comment]
Any chance you have two of the UI app on your screen?

**Comment by Acemcbutterpants** [comment]
No, I don’t.

**Comment by ThatTonybo** [comment]
Clear your cache and see if that solves it. Had another user with a similar issue and a cache clear helped.

**Comment by Acemcbutterpants** [comment]
How would I go about doing that? I could search it up but I don’t know if I will be clearing the right cache.

**Comment by ThatTonybo** [comment]
![image](https://user-images.githubusercontent.com/28176227/179173130-adc61a4b-443a-4d87-b22b-c668ce36389b.png)
![image](https://user-images.githubusercontent.com/28176227/179173167-c34994f2-0e0f-4446-8972-2997e15b3791.png)
![image](https://user-images.githubusercontent.com/28176227/179173245-dc21df0f-9bae-4423-bfe5-f3b36ff4e90e.png)

Just a standard cache clear should be fine.

**Comment by Acemcbutterpants** [comment]
Okay, I'll see if that works.

**Comment by Acemcbutterpants** [comment]
![image](https://user-images.githubusercontent.com/108896392/179262510-43d13e22-68a3-483e-9dd7-c2bcdfb8f6e1.png)
Still invisible. I tried re downloading it again from the repo this time, and its still invisible, and when i went to check if the visibility was set to invisible because i reinstalled it, i found that all of the values were gone. Should I try redownloading it from the website instead of in-game?

**Comment by ThatTonybo** [comment]
May be worth a shot. I'm not really sure what's up; just to check, could you also try removing all your UI apps, then re-add only Beam Advisor and see what happens?

**Comment by Acemcbutterpants** [comment]
Sure, I’ll try that at some point soon.

**Comment by Acemcbutterpants** [comment]
No luck even after deactivating all mods (except beam advisor of course) instead of just the UI apps I had. local://local is still empty.
Also, I have a new error message, “app not found: beamAdvisor”
If beam advisor was missing, that would explain the missing values in local://local, and beam advisor not working.
I guess it’s time to reinstall beam advisor again.

**Comment by Acemcbutterpants** [comment]
Okay, I reinstalled beam advisor with no other mods and it’s finally working. I’ll try slowly adding my mods back to see which mod is breaking it.

**Comment by ThatTonybo** [comment]
Glad to hear it's working now. Best of luck with adding the rest of the mods. I'll close this if you have no more issues?

**Comment by Acemcbutterpants** [comment]
Sure.

**Comment by Acemcbutterpants** [comment]
I did my testing and I found that Beam Advisor will not work with BeamMP. The mod for it, shown here:
![image](https://user-images.githubusercontent.com/108896392/179763837-88c1ac63-c6a4-4a91-942a-a0140980db1a.png)
must be disabled. It will automatically enable when you launch BeamNG via the MP launcher, so you will have to disable it once you are finished to use the advisor again.
You should add this to the repo page so that people know that Beam Advisor doesn't work with BeamMP.

**Comment by ThatTonybo** [comment]
I'm not really sure about that. Beam Advisor works fine with BeamMP to my knowledge, a few servers have installed it without issue. I don't actively support MP as I don't play it, but will try look into it.

**Comment by Acemcbutterpants** [comment]
I tried disabling all my mods instead of beam advisor, and adding them back one by one to see which one breaks beam advisor, and when I added back beammp and restarted the game, beam advisor was gone. I disabled it, restarted the game, and beam advisor was back. You do have to make sure that beammp is correctly enabled or disabled first though, as it sometimes likes to enable itself even after you turn it off.
